README.md

```diff
@@ -1 +1,67 @@
-# Hello-Microverse
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
```

Comment:
- [ ] Great job adding the README file 👍! As requested by the previous code review , it could be better to have all the headers and sections in the Microverse [README template.](https://github.com/microverseinc/readme-template) This is important to give detailed information about your project. 🥺 it will make your README template incredibly descriptive 😃